draft-ietf-core-dns-over-coap.md

```diff
@@ -499,6 +507,11 @@ harden against injecting spoofed responses.
 Consequently, it is of little concern to leverage the benefits of CoAP caching by setting the ID to
 0.
 
+General CoAP security considerations apply.
+Exceeding those in {{Section 11 of RFC7252}},
+the request patterns of DoC make it likely that long-lived security contexts are maintained:
+{{amp-0rtt}} goes into more detail on what can and needs to be done
+when those are resumed from a new address.
```

Comment:
With the opening "General CoAP security considerations apply." wouldn't it make more sense to make this the first paragraph?

Reply:
Moved to the top.